proto/one/enterprise/data/post_filtering.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+import "post_filtering_column_definition.proto";
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
+
+message PostFilteringTable {
+
+    // Table name.
+    string name = 1;
+
+    // Column definitions.
+    repeated PostFilteringColumnDefinition columnDefs = 2;
+
+    // ag-Grid filters JSON
+    google.protobuf.StringValue filters = 3;
```

Comment:
This looks a little awkward since it is a single string but called filters plural, perhaps filterJson would be a good name?

Reply:
It is a JSON object created by agGrid that contains dynamic objects for each of the columns in the view. It's referred to in agGrid as 'filters'.